src/Discovery/DiscoveryDetails/Components/FieldGrouping/TabField/DataDownloadList/Utils/ProcessData.test.ts

```diff
@@ -38,4 +40,13 @@ describe('ProcessData function', () => {
     expect(consoleLogSpy).toHaveBeenCalledWith('Item without title or file_name:', { other: 'Other1' });
     expect(consoleLogSpy).toHaveBeenCalledWith('Item without title or file_name:', { description: 'Description2' });
   });
+
+  it(`filters and processes large data (more than ${MAX_NUMBER_OF_ITEMS_IN_LIST} entries)`, () => {
+    const sourceFieldData = [Array.from({ length: 210 }, (_, i) => ({ title: `Title${i}`, description: `Description${i}` }))];
```

Comment:
Consider replacing hard coded value 210 for length with 

Array.from({ length:  MAX_NUMBER_OF_ITEMS_IN_LIST + 1 }

This will prevent the test from failing and needing updating if we ever change the MAX_NUMBER_OF_ITEMS_IN_LIST for some reason.